core/domain/domain.core.yml

```diff
@@ -138,6 +143,27 @@ slots:
   has_assistance:
     type: unfeaturized
 
+  daily_ci_enroll__do_enroll:
+    type: unfeaturized
+
+  daily_ci_enroll__validation_code_reference:
+    type: unfeaturized
+
+  daily_ci_enroll__validation_code:
+    type: unfeaturized
+
+  daily_ci_enroll__phone_number_error_counter:
+    type: unfeaturized
+    initial_value: 0
+
+  daily_ci_enroll__validation_code_error_counter:
+    type: unfeaturized
+    initial_value: 0
+
+  daily_ci_enroll__wants_cancel:
+    type: unfeaturized
+    initial_value: false
+
```

Comment:
If you look at other flows, there are two underscores after daily_ci. We should this convention for the slots and the messages.